content/en/docs/concepts/windows/_index.md

```diff
@@ -1,4 +1,34 @@
 ---
 title: "Windows in Kubernetes"
-weight: 50
+simple_list: true
+weight: 200 # late in list
+description: >-
+  Kubernetes supports nodes that run Microsoft Windows.
 ---
+
+Kubernetes supports worker {{< glossary_tooltip text="nodes" term_id="node" >}}
+running either Linux or Microsoft Windows.
+
+{{% thirdparty-content single="true" %}}
+
+The CNCF and its parent the Linux Foundation take a vendor-neutral approach
+towards compatibility: the Kubernetes project produces open source software
```

Comment:
Not sure if this tone is correct. "the Kubernetes project produce things to make Windows worker node a reality" ... is it accurate? I'm afraid there are things produced by the wider CNCF community and some adjacent communities.